README.md

```diff
@@ -71,10 +63,43 @@ be good to go.
 Building the front-end
 ----------------------
 
-Run `grunt build` to regenerate the front-end logic and css files. For production, 
+Run `grunt build` to regenerate the front-end logic and css files. For production,
 run with `grunt build:prod` to get minified versions, and run `grunt build:prod:true` to enable
 auto building whenever a front-end file changes. See `Gruntfile.js` for details.
 
 **NOTE:** Our front-end is in transition from an older architecture using Requirejs to
 a new one using our build system. Check in our IRC channel (irc.mozilla.org#thimble) for
 the latest state of development.
+
+Bramble code editor
+-------------------
+
+In order to load Thimble, the bramble code editor must be statically served separately.
+The simplest way to do this is to:
+
+1) Clone the editor (outside your cloned Thimble source) and checkout to the `bramble` branch
+
+```
+git clone https://github.com/humphd/brackets.git --recursive
+```
+
+2) Install all dependencies (you will need to be in the brackets directory)
+
+```
+npm install && git submodule update --init
+```
+
+3) Serve the root folder of this repo statically. Thimble will look for it on
+localhost(port 8000 by default). Here are some examples:
+
+```
+python -m SimpleHTTPServer
+```
+
+   or
+
+```
+npm install -g live-server && live-server --port=8000
+```
+
+**NOTE:** Pay special attention to the ENV variable in Thimble for this (`BRACKETS_URI`), since it must point at the url of the local server you run to serve the brackets resources. By default, this would be [http://localhost:8000](http://localhost:8000)
```

Comment:
Nit: *Pay special attention to thimble's `BRACKETS_URI` ENV variable, since it must point at the URL of the local server you created in step 3*